src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/InventaryService.java

```diff
@@ -1,24 +1,51 @@
 package com.riwi.localstorage.riwi_local_storage.infrastructure.services;
 
+import java.util.Date;
 import java.util.Optional;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 
 import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.InventaryRequest;
 import com.riwi.localstorage.riwi_local_storage.api.dto.request.update.InventaryRequestUpdate;
 import com.riwi.localstorage.riwi_local_storage.api.dto.response.InventaryResponse;
+import com.riwi.localstorage.riwi_local_storage.domain.entities.Inventory;
+import com.riwi.localstorage.riwi_local_storage.domain.repositories.InventoryRepository;
 import com.riwi.localstorage.riwi_local_storage.infrastructure.abstract_services.IInventaryService;
+import com.riwi.localstorage.riwi_local_storage.infrastructure.mappers.inventoryMappers.InventoryCreateMapper;
+import com.riwi.localstorage.riwi_local_storage.util.exeptions.InventoryExpirationDatePassed;
+
+
 
 import lombok.AllArgsConstructor;
 
 @Service
 @AllArgsConstructor
-public class InventaryService implements IInventaryService {@Override
+public class InventaryService implements IInventaryService {
+    @Autowired
+    private final InventoryRepository repository;
+
+    @Autowired
+    private final InventoryCreateMapper createMapper;
+
+    /*--------------------
+     * CREATE INVENTORY
+     * -------------------
+     */
+
+    @Override
     public InventaryResponse create(InventaryRequest request) {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'create'");
+        Date fechaActual = new Date();
```

Comment:
Date fechaActual = new Date();

Remember that variables must be declared in English and in camelCase format.

Reply:
I've already made the updates